src/cryptography/x509/extensions.py

```diff
@@ -862,11 +844,7 @@ def __init__(self, usages):
 
         self._usages = usages
 
-    def __iter__(self):
-        return iter(self._usages)
-
-    def __len__(self):
-        return len(self._usages)
+    __len__, __iter__, __getitem__ = _make_sequence_methods("_usages")
```

Comment:
So we didn't previously implement `__getitem__` on EKU?

Reply:
Yeah :-/ I decided this was simpler.